client/javascripts/components/shortlist/shortlist.tag

```diff
@@ -0,0 +1,64 @@
+<shortlist>
```

Comment:
Following the other components, and site wide style, indentation should be of 4 spaces.

Reply:
done.